### PR TITLE
Add column mv v3/add column

### PR DESCRIPTION
**Pull request by albertojuan**
Example and documentation of the steps required to add a column to a Materialized View

### CODE REVIEW

File: v3/add_new_column_to_a_materialized_view_v3/README.md [hunk 1]
```diff
@@ -1,3 +1,76 @@
...
+
+This change needs to re-create the Materialized View and populate it again with all the data without stoping our ingestion.
+
+For that the steps will be:
+
+(In this Pull Request)
+1. Change the Materialized View (Pipe and Data Source) to add the new column.
+2. Bump version from 0.0.0 -> 0.0.1. It will create a new Preview Release internally forking the Materialized View and its dependencies.
+
+(Out of this Pull Request)
```
Comment (alrocar) [inline v3/add_new_column_to_a_materialized_view_v3/README.md:15]: Let's include it in the README.md so it's reproducible

**Review by alrocar — COMMENTED** [review]
Thanks! Much cleaner ✨

File: v3/add_new_column_to_a_materialized_view_v3/README.md (hunk 1 above)
Comment (albertojuan) [reply v3/add_new_column_to_a_materialized_view_v3/README.md]: Changed, you can review again ;)

File: v3/add_new_column_to_a_materialized_view_v3/README.md [hunk 2]
```diff
@@ -1,3 +1,93 @@
...
+Once you have deploy the previous changes and are ready in a Preview Release you need to backfill the data previous to the Materialized View re-creation.
+
+- Get the creation date by executing the following command
+```sh
+tb --semver 0.1.0 datasource ls
+```
+or
+
+```sh
+tb --semver 0.1.0 sql "select timestamp from tinybird.datasources_ops_log where event_type = 'create' limit 1"
```
Comment (alrocar) [inline v3/add_new_column_to_a_materialized_view_v3/README.md:78]: we could `order by timestamp desc limit 1`

**Review by alrocar — APPROVED** [review]
minor comment. Good job 💪

File: v3/add_new_column_to_a_materialized_view_v3/README.md (hunk 2 above)
Comment (albertojuan) [reply v3/add_new_column_to_a_materialized_view_v3/README.md:78]: Good one, for this very specific example should be the same but I will add it for clarity and avoid leading to errors in different scenarios. In addition, I will add another condition with the data source name.

Comment (albertojuan) [reply v3/add_new_column_to_a_materialized_view_v3/README.md]: [Done!](https://github.com/tinybirdco/use-case-examples/pull/141/files#diff-85a735bae4fc0206fe9170b2f442eb4d78324f9dc10e6ae2d74812d9fc7be4ceR78)

**Review by alrocar — APPROVED** [review]

**Pull request merged**